tool/bindings_generator/generate_bindings.dart

```diff
@@ -7,11 +7,48 @@ import 'dart:js_util';
 
 import 'translator.dart';
 import 'util.dart';
+import 'webref_css_api.dart';
 import 'webref_idl_api.dart';
 
+/// Generate CSS property names for setting / getting CSS properties in JS.
+Future<List<String>> _generateCSSStyleDeclarations() async {
+  final cssStyleDeclarations = <String>{};
+  final array = objectEntries(await promiseToFuture<JSObject>(css.listAll()));
+  for (var i = 0; i < array.length.toDart; i++) {
+    final entry = array[i.toJS] as JSArray;
+    final data = entry[1.toJS] as JSObject;
+    final properties = getProperty<JSArray?>(data, 'properties');
```

Comment:
any reason why not add a couple static-interop classes to represent the array entries and property entries?